examples/01_plotting/plot_transparency.py

```diff
@@ -0,0 +1,295 @@
+r"""
+Plotting images with transparent thresholding
+=============================================
+
+Standard thresholding means that any data below a threshold value
+is completely hidden from view.
+However, "transparent thresholding" allows
+for the same suprathreshold results to be observed,
+while also showing subthreshold information
+with an opacity that fades with magnitude
+(:footcite:t:`Allen2012`, :footcite:t:`Chen2022`,
+:footcite:t:`Taylor2023`, :footcite:t:`Sundermann2024`).
+
+This makes use of the "alpha" value that overlays can have
+when using some plotting functions (like :func:`matplotlib.pyplot.imshow`).
+This "alpha" value goes from 0 (perfectly transparent) to 1 (perfectly opaque).
+
+Consider having an underlay color :math:`RGB_{ulay}`
+and overlay color :math:`RGB_{olay}`,
+where a threshold value T is applied
+and we consider how an element (voxel, node...) with value M is shown.
+
+"opaque" thresholding
```

Comment:
The small case for the first word looks a bit weird to me in the rendered version
```suggestion
"Opaque" thresholding
```